src/jest.js

```diff
@@ -47,7 +47,15 @@ function getTestPaths(testRunner, config, patternInfo) {
   if (patternInfo.onlyChanged) {
     return findOnlyChangedTestPaths(testRunner, config);
   } else {
-    return testRunner.promiseTestPathsMatching(new RegExp(patternInfo.pattern));
+    return new Promise((resolve, reject) => {
+      fs.access(patternInfo.pattern, (err) => {
+        resolve(
+          err?
```

Comment:
ohh, code style: this needs a space and the colon on the line below also needs a space before :)